modules/administration-guide/partials/proc_configuring-external-PostgreSQL.adoc

```diff
@@ -22,10 +22,8 @@ ALTER USER <database-user> WITH SUPERUSER
 +
 [subs="+quotes,+attributes"]
 ----
-CREATE USER <identity-database-user> WITH PASSWORD '<identity-database-password>' <1> <2>
-CREATE DATABASE <identity-database>                                               <3>
-GRANT ALL PRIVILEGES ON DATABASE <identity-database> TO <identity-database-user>
+CREATE USER keycloak WITH PASSWORD '<identity-database-password>' <1>
```

Comment:
@tolusha just to make sure - these instructions supposed to work for deployment with the **internal** keycloak and **external** (e.g. RDS) db (technically there would be 2 db one for Che and one for keycloak) ?

Reply:
@ibuziuk 
right, external Db and internal keycloak server

Reply:
This PR fix creation keycloak DB since its name and user are hardcoded in che-operator and its not possible to configure them for now.